authmate/authmate.go

```diff
@@ -384,6 +384,26 @@ func buildContext(rules []byte) ([]*session.ContainerContext, error) {
 		if err != nil {
 			return nil, fmt.Errorf("failed to unmarshal rules for session token: %w", err)
 		}
+
+		var (
+			containsPut     = false
+			containsSetEACL = false
+		)
+		for _, s := range sessionCtxs {
+			if s.ToV2().Wildcard() {
```

Comment:
I'm not sure if we need this condition. As I understand `PUT` is valid only for wildcard so we can drop this condition and invoke `s.ForPut()` next to `containsPut = true` (line 395)